ietf/sync/iana.py

```diff
@@ -69,7 +69,7 @@ def fetch_changes_json(url, start, end):
     username = "ietfsync"
     password = settings.IANA_SYNC_PASSWORD
     headers = { "Authorization": "Basic %s" % force_str(base64.encodebytes(smart_bytes("%s:%s" % (username, password)))).replace("\n", "") }
-    text = requests.get(url, headers=headers).text
+    text = requests.get(url, headers=headers, timeout=settings.DEFAULT_REQUESTS_TIMEOUT).text
```

Comment:
The addition of the timeout opens the possibility of raising a requests.exceptions.Timeout exception. It would be nice to handle that, though it would be understandable if you decide that erroring out of the command line utility is a correct behavior.